chsdi/views/features.py

```diff
@@ -291,10 +291,7 @@ def _identify_grid(params, layerBodIds):
 
     pointCoordinates = geometry.coordinates
 
-    # TODO change bucket and profile names
-    # To work with pserve use your own profilename
-    testProfilename = params.request.registry.settings['boto_test_profilename']
-    bucket = get_bucket(profile_name=testProfilename, bucket_name='waf-wmts-test')
+    bucket = get_bucket(profile_name='chsdi_aws_admin_ro', bucket_name='mwks6dv2y5dsbbgg-vectortiles')
```

Comment:
@danduk82 @loicgasser The idea is to dispatch this profile on all our mf0 machines?